### ISSUE TITLE
Delete line shortcut will focus on the previous line instead of on the next line after deletion

### ISSUE THREAD

**Issue by matejdro** [timhor/obsidian-editor-shortcuts#15]
In VS Code and IntelliJ, after deleting a line, caret will be put on the following line. But with this plugin, caret will move one line up.

Could this behavior be fixed to mimic other text editors?

**Comment by timhor** [comment]
I'm not seeing the behaviour where the cursor moves one line up. Could you please provide an example where it's doing this?

**Comment by matejdro** [comment]
It looks like I had an old version of the plugin. Updating from 1.2.1 to 1.4.0 fixed it.

**Comment by vanbang9710** [comment]
I also face this issue
When putting a cursor here
 ![|50%](https://i.imgur.com/zGNu2gA.png)
Note:
```
updated: [2023-04-02T22:48:14](https://hyp.is/xy3AdtFtEe2MiXtkpn06YA/www.thegioididong.com/game-app/top-7-phan-mem-web-kiem-tra-dao-van-online-mien-phi-chinh-xac-1312510)  
highlight:: DoIT là phần mềm kiểm tra đạo văn trực tuyến và là sản phẩm của Trường Đại học Công nghệ thuộc Đại học Quốc gia Hà Nội  
annotation:: [a](http://doit.uet.vnu.edu.vn/)
```

**Comment by timhor** [comment]
That one I can reproduce, will look into it. Appears to happen only when the line is long enough to be wrapped and your cursor is on the non-wrapped portion.

**Comment by timhor** [comment]
This should be fixed in [v1.13.1](https://github.com/timhor/obsidian-editor-shortcuts/releases/tag/1.13.1)
